src/main/groovy/de/dkfz/roddy/execution/jobs/direct/synchronousexecution/DirectCommand.groovy

```diff
@@ -61,22 +61,22 @@ class DirectCommand extends Command {
             parameterBuilder << parameters.collect { key, value -> "${key}=${value}" }.join(" ")
         }
 
-        //TODO Log handling
+        // Dependencies are ignored. Direct commands are executed in-sync.
 
-        //TODO email handling? Better not
-
-        //Dependencies are ignored
+        // Processing commands are ignored BE does not offer job scheduling on its own.
 
-        //Grouplist is ignored
+        //TODO email handling? Better not
 
-        //Umask is ignored
+        //TODO Grouplist is ignored
 
-        //Processing commands are ignored
+        //TODO Umask is ignored
 
         //TODO Command assembly should be part of the file system provider? Maybe there is a need for a local file system provider?
         //This is very linux specific...
-        commandString << parameterBuilder.toString() << StringConstants.WHITESPACE << command; // << job.configuration.getProcessingToolPath(executionContext, "wrapinScript").getAbsolutePath();
+        commandString << parameterBuilder.toString() << StringConstants.WHITESPACE << command << " &> ${job.jobLog.out} & wait &> /dev/null";
```

Comment:
I am wondering now, whether it would be a good idea to provide a `job.jobLog.getOut(String jobID)` method in BatchEuphoria. The string-replacement below is otherwise repeated in every Command implementation.

The "{JOB_ID}" string is also already defined as constant in JobLog. Please use the constant or even better implement the error(String jobID) and out(String jobID) methods in BE.